pkg/executive/db_limiter.go

```diff
@@ -114,7 +114,7 @@ func (l *dbLimiter) checkWriterRates(ctx context.Context, tx *sql.Tx, lr limiter
 	}
 	writerLimit := l.limitForWriter(lr.writerName)
 	allowed := amount <= writerLimit
-	events.Debug("limiter: writer:%v writerLimit:%v amount:%v allowed:%v", lr.writerName, writerLimit, amount, allowed)
+	log.Debug("limiter", "writer", lr.writerName, "writerLimit", writerLimit, "amount", amount, "allowed", allowed)
```

Comment:
How does this render in comparison to the original output?

Reply:
Top and bottom 

```
{"time":"2024-02-26T15:16:38.977843-08:00","level":"DEBUG","source":{"function":"main.main","file":"/Users/kburke/src/github.com/kevinburkesegment/go-experiments/events-log/main.go","line":31},"msg":"limiter","data":{"writer":"test-writer","writerLimit":17,"amount":19,"allowed":43}}
{"level":"DEBUG","time":"2024-02-26T15:16:38.97784-08:00","info":{"source":"events-log/main.go:30"},"data":{},"message":"limiter: writer:test-writer writerLimit:17 amount:19 allowed:43"}
```